flomio/passninja-js
# Surface error when create pass method is invoked with missing required keys

# Change Log
* Correct delete method documentation in README per Sebastian's suggestions
* Add passtypes/keys check when creating a pass, missing required keys surfaces an error

# Verification
* Confirm that delete method docs are corrected
* Confirm error is surfaced when create pass method is invoked with missing keys

Issue: flomio/passninja-js#4
